js/components/Leaderboard/index.js

```diff
@@ -21,9 +21,6 @@ class Leaderboard extends React.Component {
 			filter: {
 				campaign_id: '7536',
```

Comment:
Did the `campaign_id` change when we killed runs?

Reply:
nope! Double checked this! :)

Reply:
Oh, good flag! Does [this](https://activity.dosomething.org/campaigns/7536) look like the right campaign?